src/System.DirectoryServices.AccountManagement/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -1,4 +1,23 @@
-<!-- Temporary workaround to get build to work. -->
-credui.dll!CredUIParseUserNameW
-advapi32.dll!LsaNtStatusToWinError
 activeds.dll!ADsOpenObject
```

Comment:
One way or another I'll disable it for libraries for whom we dont' care about UWP